test/gtest/ucp/test_ucp_wakeup.cc

```diff
@@ -346,22 +344,10 @@ test_ucp_wakeup_events::do_tx_rx_events_test(const std::vector<std::string>& tra
     ucp_worker_flush(p_entity[1]->worker());
 }
 
-void
-test_ucp_wakeup_events::setup_worker_param_events(ucp_worker_params_t& params,
-                                                  unsigned events)
-{
-    params.field_mask |= UCP_WORKER_PARAM_FIELD_EVENTS;
-    params.events = events;
-}
-
 UCS_TEST_P(test_ucp_wakeup_events, events)
 {
     UCS_TEST_SKIP_R("Functionality is not implemented yet");
-
-    EXPECT_TRUE(GetParam().worker_params.field_mask &
-                UCP_WORKER_PARAM_FIELD_EVENTS);
-    do_tx_rx_events_test(GetParam().transports,
-                         GetParam().worker_params.events);
+    // do_tx_rx_events_test(GetParam().transports, GetParam().variant);
```

Comment:
Commented code easy to miss during refactoring and hardly restore later. Pls uncomment this or remove the test case if not needed anymore.

Reply:
actually, you added this test which is always skipped, and coverity started complaining about this.
can we remove the whole test or need to un-skip it?

Reply:
we don't have implementation to un-skip, lets remove then add new one when feature is implemented.

Reply:
ok